packages/protocol/engine/consensus/acceptance/gadget.go

```diff
@@ -158,29 +221,40 @@ func (a *Gadget) block(id models.BlockID) (block *Block, exists bool) {
 	return storage.Get(id)
 }
 
-func (a *Gadget) propagateAcceptance(marker markers.Marker) (queuedBlocks []*Block) {
+func (a *Gadget) propagateAcceptance(marker markers.Marker, confirmed bool) (acceptedBlocks, confirmedBlocks []*Block) {
```

Comment:
`propagateAcceptance` is not accurate. It should be `propagateAcceptanceConfirmation`